apps/hobbies-helsinki/i18nRoutes.config.js

```diff
@@ -20,8 +20,8 @@ const i18nRoutes = {
     { source: '/sidor/:slug*', locale: 'sv' },
   ],
   '/accessibility': [
-    { source: '/saavutetavuusseloste', locale: 'fi' },
-    { source: '/tillgäntglighetsutlåtande', locale: 'sv' },
+    { source: '/saavutettavuusseloste', locale: 'fi' },
+    { source: '/tillgänglighetsutlåtande', locale: 'sv' },
```

Comment:
`/accessibility` and `/about` can be totally removed. See the another comment about these.

Reply:
Removed in 8de789f1ef34ec53de94241e3c6717a413e761f2